### ISSUE TITLE
still about training

### ISSUE THREAD

**Issue by 37integer** [SerezD/NVAE-from-scratch#4]
sorry bother you again, it's like result still looks like very weird, i see the loss it's seems not collapse but still the result of image looks like very weird. do you have any idea about this?
<img width="881" alt="5" src="https://github.com/user-attachments/assets/0b42294f-6b6b-4ecd-a2b1-1c5d2cfa07ee">

<img width="880" alt="6" src="https://github.com/user-attachments/assets/681d4825-7ceb-4743-b851-b2b9d6250a5a">

the config i use is:
```
resolution:
  - 3
  - 256
  - 256

autoencoder:
  num_scales: 5 
  num_groups_per_scale: 16 
  is_adaptive: true
  min_groups_per_scale: 4 
  num_cells_per_group: 2 
  num_latent_per_group: 20
  num_nf_cells: 2
  initial_channels: 30
  num_pre-post_process_blocks: 1 
  num_pre-post_process_cells: 2
  num_logistic_mixtures: 10 

training:
  seed: 0
  epochs: 500
  warmup_epochs: 5
  cumulative_bs: 16
  base_lr: 1e-3
  min_lr: 1e-4
  weight_decay: 3e-4
  eps: 1e-3
  spectral_regularization:
    weight_decay_norm: 1e-2
    weight_decay_norm_anneal: True
    weight_decay_norm_init: 1.
  kl_anneal:
    kl_anneal_portion: 0.3 
    kl_const_portion: 0.0001 
    kl_const_coeff: 0.0001
```

**Comment by SerezD** [comment]
I think that your main issue right now may be the small batch size you are using, which prevents the learning of strong generalizations at a generation level. You can try to reduce the learning rate a little bit more and increase the number of epochs to see if that helps. 
In general, I am not sure you can obtain good results by training such a large model with small computation capabilities.
